### PR TITLE
HVG-759: Embark, Passage transitions

### PR DESCRIPTION
**Pull request by hedvigoscar**
### Checklist

- [ ] Functionality is covered by an integration test
- [ ] Functionality is accessible in engineering mode

### CODE REVIEW

File: app/src/main/java/com/hedvig/app/feature/embark/EmbarkViewModel.kt [hunk 1]
```diff
@@ -234,9 +241,11 @@ abstract class EmbarkViewModel : ViewModel() {
             }
         )
```
Comment (hugokallstrom) [inline app/src/main/java/com/hedvig/app/feature/embark/EmbarkViewModel.kt:243]: Remove commented code

**Review by hugokallstrom — APPROVED** [review]

File: app/src/main/java/com/hedvig/app/feature/embark/EmbarkViewModel.kt [hunk 2]
```diff
@@ -234,9 +241,11 @@ abstract class EmbarkViewModel : ViewModel() {
...
 
+/*
         val passagesLeft = getPassagesLeft(passageWithMessage)
         val progress = Percent((((totalSteps.toFloat() - passagesLeft.toFloat()) / totalSteps.toFloat()) * 100).toInt())
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/feature/embark/EmbarkViewModel.kt:246]:
```suggestion
```

Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/feature/embark/EmbarkViewModel.kt:246]:
```suggestion
```

File: app/src/main/java/com/hedvig/app/feature/embark/EmbarkViewModel.kt [hunk 3]
```diff
@@ -234,9 +241,11 @@ abstract class EmbarkViewModel : ViewModel() {
...
-        return CurrentPassageWithProgress(passageWithMessage, progress)
+*/
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/feature/embark/EmbarkViewModel.kt:247]:
```suggestion
```

**Review by hugokallstrom — APPROVED** [review]

**Pull request merged**